src/boot_semver/core.clj

```diff
@@ -90,4 +93,9 @@
       (when (and (nil? (:no-update *opts*)) (not= curver version))
         (util/info (clojure.string/join ["Updating Project Version...: " curver "->" version "\n"]))
         (set-version! semver-file version))
-      fs)))
+      (if (:include *opts*)
+        (-> fs (boot/add-resource
```

Comment:
I am working on a boot-new generator which will generate a custom namespace with version information for use directly in projects. These two options should solve pretty much all the version use cases I can think of.